sites/geohub/src/lib/server/helpers/createDatasetLinks.ts

```diff
@@ -5,13 +5,16 @@ import { generateAzureBlobSasToken } from './generateAzureBlobSasToken';
 export const createDatasetLinks = async (
 	feature: DatasetFeature,
 	origin: string,
-	titilerUrl: string
+	titilerUrl: string,
+	tool?: string
```

Comment:
Is this `tool` param necessary in this function? `tool (algorithm ID)` can be fetched from tags like what you are doing in line 14-17

Reply:
@JinIgarashi Some datasets might hold more than one tool. So in this case, would it be advisable to have tool as a tag? Fetching from tags might not work properly then I believe